README.md

```diff
@@ -80,9 +80,7 @@ A promise that resolves to the record if found, null if not found, and rejects i
 | :-- | :-- | :-- |
 |config | Object | Each of its properties defines your get operation |
 |config.table | String | The datastore table name |
-|config.params| Object | Each of its properties defines the get parameters |
-|config.params.id| String |  The ID that the data is associated with |
-|config.params.data| Object | The data that will be saved in the datastore |
+|config.params| Object | The data that will be saved in the datastore |
```

Comment:
You don't have to change this?
https://github.com/screwdriver-cd/datastore-base/blob/32030af16912d6556f9a5db3007d54117b469c21/index.js#L68-L69